samples/drivers/ht16k33/nrf52840_pca10056.overlay

```diff
@@ -12,7 +12,7 @@
 		reg = <0x70>;
 		label = "HT16K33";
                 /* Uncomment to use IRQ instead of polling: */
-		/* irq-gpios = <&gpio1 8 (GPIO_PUD_PULL_UP | GPIO_INT_ACTIVE_LOW)>; */
+		/* irq-gpios = <&gpio1 8 GPIO_PUD_PULL_UP>; */
```

Comment:
The (undocumented) consensus seems to be that active level should be made explicit, so this should add `GPIO_ACTIVE_HIGH`.  Though that's not consistent with normal use of pull-up, so maybe it should be `GPIO_ACTIVE_LOW`.  Except in the driver the interrupt is configured for edge-to-inactive.

Reply:
@pabigot My intention was to to use an IRQ which triggers on falling edges. I have updated it to explicitly specify the active level.